src/components/issue-description.component.js

```diff
@@ -15,13 +15,6 @@ import { translate, relativeTimeToNow } from 'utils';
 import { colors, fonts, normalize } from 'config';
 import { v3 } from 'api';
 
-const styles = StyleSheet.create({
```

Comment:
why was this removed?

Reply:
It was used just for the Badge styles so I removed it. I don't think is needed here, also styled-components are already imported.
Also the styles defined for the Badge don't change the UI so they are useless for what I can saw.